### PR TITLE
Replace multi-line code block tags with blockquote

### PR DESCRIPTION
**Pull request by erictleung**
Similar to #3473, but keeping that open because that issue is about single lines of code that overflow. However, this PR was brought about in discussions in #3473.

Tested locally.

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/html5-and-css.json [hunk 1]
```diff
@@ -808,13 +789,7 @@
...
         "CSS borders have properties like <code>style</code>, <code>color</code> and <code>width</code>",
         "For example, if we wanted to create a red, 5 pixel border around an HTML element, we could use this class:",
-        "<code>&#60;style&#62;</code>",
-        "<code>&nbsp;&nbsp;.thin-red-border {</code>",
-        "<code>&nbsp;&nbsp;&nbsp;&nbsp;border-color: red;</code>",
-        "<code>&nbsp;&nbsp;&nbsp;&nbsp;border-width: 5px;</code>",
-        "<code>&nbsp;&nbsp;&nbsp;&nbsp;border-style: solid;</code>",
-        "<code>&nbsp;&nbsp;}</code>",
-        "<code>&#60;/style&#62;</code>",
+        "<blockquote>&#60;style&#62;<br>&nbsp;&nbsp;.thin-red-border {<br>&nbsp;&nbsp;&nbsp;&nbsp;border-color: red;<br>&nbsp;&nbsp;&nbsp;&nbsp;border-width: 5px;<br>&nbsp;&nbsp;&nbsp;&nbsp;border-style: solid;<br>&nbsp;&nbsp;}</code>&#60;/style&#62;</blockquote>",
```
Comment (SaintPeter) [inline seed/challenges/01-front-end-development-certification/html5-and-css.json]: There should be a `<br>` before `</style>` at the end of this block.  Also, there is a spurious closing `</code>` tag at the end of the line.

**Comment by SaintPeter** [comment]
This looks really good.  Just one minor change noted above and we're good to go.

**Comment by erictleung** [comment]
Thanks for double checking. Almost got all of them! I hope the final changes fix them. And thanks for making the `<blockquote>` tag! Looks great now.

**Comment by SaintPeter** [comment]
Solid.  I think these look a lot better now.

**Pull request merged**